modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/hooks/usePage.es.js

```diff
@@ -19,6 +19,7 @@ const PageContext = React.createContext({});
 
 export const PAGE_TYPES = {
 	CHANGE_ACTIVE_PAGE: 'activePageUpdated',
+	DND_DROP: 'fieldDrop',
```

Comment:
Maybe `FIELD_DROPPED` or `FIELD_DROP` would be more inline with the pattern of the other events?

Reply:
Hmm, it makes sense. I'll replace that.